apps/web/public/static/locales/en/common.json

```
@@ -2486,6 +2486,8 @@
   "unable_to_subscribe_to_the_platform": "An error occurred while trying to subscribe to the platform plan, please try again later",
   "updating_oauth_client_error": "An error occurred while updating the OAuth client, please try again later",
   "creating_oauth_client_error": "An error occurred while creating the OAuth client, please try again later",
+  "event_type_colour": "Event type colour",
```

```suggestion
  "event_type_color": "Event type color",
```
We should use American English

---

done